appside/test_frames.html

```diff
@@ -38,7 +38,9 @@
   <script src="frames/summary.js"></script>
   <script src="frames/intro.js"></script>
   <script src="frames/emotion_selection.js"></script>
+  <script src="frames/likert.js"></script>
   <script src="debug/tools.js"></script>
+  <script src="data/constants.js"></script>
```

Comment:
can be removed after implementing changes suggested above

Reply:
👍